src/storage/posix/storage.c

```diff
@@ -42,55 +42,23 @@ struct StoragePosix
     MemContext *memContext;                                         // Object memory context
 };
 
-/**********************************************************************************************************************************/
-static bool
-storagePosixExists(THIS_VOID, const String *file, StorageInterfaceExistsParam param)
```

Comment:
I think removing the Exists() interface functions is a bit of a mistake. These are more streamlined because checking existence should not care about type, size, timeModified but now we are setting these when we don't need to and making exceptions in the code. I think if you had implemented the storageInfoLevelExists in the *Info functions to replace the *Exists functions I'd feel differently, but I think all we've done is make things more confusing for no gain.